### ISSUE TITLE
XUL apps generate files in appdata dir

### ISSUE THREAD

**Issue by almarklein** [zoofIO/flexx#150]
A xul app produces "config" files in the appdata directory. Since we essentially create a new app (with a new app id) every time a xul app is launched, this generates a vast amount of data over time. Each entry is about 10MB.

On Windows, the data is placed in `C:\Users\almar\AppData\Roaming\None\`

We need to either, 1) prevent this from hapening; 2) clean it up; 3) stop using a different app id for each launch.

**Comment by almarklein** [comment]
Pfew! This one was nasty, and I only ran into it by accident while examining what directories were taking the most memory on my hard drive ...

**Comment by almarklein** [comment]
I just found a bunch of profile thingies in:
* `C:\Users\almar\AppData\Roaming\None` - I'm pretty sure that I cleaned this. There were not an awful lot of them here though. 
* `C:\Users\almar\AppData\Local\None` - a lot. Maybe I just never cleaned these up because earlier I only checked the roaming dir.

I cleared both dirs and then ran a Flexx app with xul/firefox/chromeapp runtimes a few times to see if either dir is populated again. They are not.

Everything is probably fine, but wanted to log this, just in case.
